Sources/SwiftSyntaxBuilder/SyntaxNodeWithBody.swift

```diff
@@ -12,6 +12,8 @@
 
 import SwiftSyntax
 
+// MARK: - Error
+
```

Comment:
Is there a reason for this mark?

Reply:
No, I ended not adding an error in this file.